### PR TITLE
Add contribute.md for the new documentation

### PR DESCRIPTION
**Pull request by ra-beer**
Signed-off-by: Ramona Beermann <ramona.beermann@osism.tech>

### CODE REVIEW

File: doc/source/contribute.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Contribute
+
+For contributing to this project, please create pull requests or issues in this GitHub
+[OpenStack Image Manager](https://github.com/osism/openstack-image-manager) repository.
+To prioritize the import of a new image [Contact Us](mailto:info@osism.tech)  with reference to the created pull request.
```
Comment (berendt) [inline doc/source/contribute.md:5]: double space before ``with``

File: doc/source/contribute.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Contribute
+
+For contributing to this project, please create pull requests or issues in this GitHub
+[OpenStack Image Manager](https://github.com/osism/openstack-image-manager) repository.
```
Comment (berendt) [inline doc/source/contribute.md:4]: Would prefer to link ``GitHub repository`` instead of only ``GitHub``.

File: doc/source/contribute.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+To prioritize the import of a new image [Contact Us](mailto:info@osism.tech)  with reference to the created pull request.
+If you want to contribute to this documentation please do the same like described above.
+
+After creating a pull request, please check the result of the GitHub Actions and correct any errors identified.
```
Comment (berendt) [inline doc/source/contribute.md:8]: GitHub Actions --> Zuul

File: .github/workflows/check-markdown-syntax.yml [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+      - '**.md'
+
+jobs:
+  check-markdown-syntax:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+      - run: npm install -g markdownlint-cli
+      - run: markdownlint -c .markdownlint.yaml .
```
Comment (berendt) [inline .github/workflows/check-markdown-syntax.yml:23]: This should be a Zuul job. Also please open a 2nd PR for this.

**Review by berendt — CHANGES_REQUESTED** [review]

File: doc/source/contribute.md (hunk 1 above)
Comment (ra-beer) [reply doc/source/contribute.md:5]: done

File: doc/source/contribute.md (hunk 2 above)
Comment (ra-beer) [reply doc/source/contribute.md:4]: The link is in the word OpenStack Image Manager.
It should shows like GitHub OpenStack Image Manager repository.
And if you klick on OpenStack Image Manager you should redirected to the right repo

File: doc/source/contribute.md (hunk 3 above)
Comment (ra-beer) [reply doc/source/contribute.md:8]: Done

File: .github/workflows/check-markdown-syntax.yml (hunk 4 above)
Comment (ra-beer) [reply .github/workflows/check-markdown-syntax.yml:23]: Will create another PR for linting via zuul

**Review by berendt — APPROVED** [review]

**Pull request merged**